app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,38 @@
+class ExpensesController < ApplicationController
+  
+  def create
+    begin
+      expense = Expense.create(expense_params)
+      render status: :created, json: expense.to_json
+    rescue
+      render status: 400
+    end
+  end
+
+  def destroy
+    begin
+      params.require(:id)
```

Comment:
Okay so I just checked and it looks like Rails automatically returns a `404` if we don't specify the `:id` in the `url`. So, we can take this out.

Reply:
ok